themes/stanford-style/lms/templates/index.html

```diff
@@ -0,0 +1,73 @@
+<%inherit file="main.html" />
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+%>
+
+<section class="home">
+  <header>
+    <div class="outer-wrapper">
+      <div class="title">
+        <hgroup>
+          % if homepage_overlay_html:
+            ${homepage_overlay_html}
+          % else:
+            <h1>${_("Free courses from <strong>{university_name}</strong>").format(university_name="Stanford")}</h1>
+            <h2>${_("For anyone, anywhere, anytime")}</h2>
+          % endif
+        </hgroup>
+        % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
+          <div class="course-search">
+            <form method="get" action="/courses">
+              <label><span class="sr">${_("Search for a course")}</span>
+                <input class="search-input" name="search_query" type="text" placeholder="${_("Search for a course")}"></input>
+              </label>
+              <button class="search-button" type="submit">
+                <i class="icon fa fa-search" aria-hidden="true"></i><span class="sr">${_("Search")}</span>
+              </button>
+            </form>
+          </div>
+        % endif
+
+      </div>
+
+      % if show_homepage_promo_video:
+        <a href="#video-modal" class="media" rel="leanModal">
+          <div class="hero">
+            <div class="play-intro"></div>
+          </div>
+        </a>
+      % endif
+    </div>
+
+  </header>
+  <%include file="${courses_list}" />
+
+</section>
+
+<section id="video-modal" class="modal home-page-video-modal video-modal">
+  <div class="inner-wrapper">
+    <%
+      youtube_video_id = "2gmreZObCY4"
```

Comment:
I am removing this for now, I will rebase this branch with master after PR #11138 is merged and resolve conflicts.